apirest.md

```diff
@@ -27,96 +27,86 @@
 
 ## Glossary
 
-Endpoint
-:   Resource available though the api.
+Endpoint: Resource available though the api.
```

Comment:
This is not OK. This will break display in GLPi, old notation displays `<dl><dt><dd>` elements, with your changes, we got only a onle line `<p>`.